bucket/tableplus.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "110",
+    "description": "Relational database explorer",
+    "homepage": "https://tableplus.com",
+    "license": "Proprietary",
```

Comment:
I would like to see URL to [license agreement](https://tableplus.com/blog/2018/01/license-agreement.html) or pricing